tests/tests_idempotency_2022.yml

```diff
@@ -8,7 +8,7 @@
     mssql_accept_microsoft_cli_utilities_for_sql_server_eula: true
     mssql_accept_microsoft_sql_server_standard_eula: true
     mssql_version: 2022
-    mssql_manage_selinux: "{{ mssql_run_selinux_confined }}"
+    mssql_manage_selinux: "{{ mssql_run_selinux_confined | d(false) }}"
```

Comment:
Yes, this will fix the error, but it might not be testing what you want to test.  I'll defer to @spetrosi but I think the intention here is to test the manage selinux functionality if mssql has the ability be run confined, which must be dynamically determined later.

Reply:
OK -- let's handle that in #350 then. On my next push I'll remove that commit here. TBH I'm very close to reverting the hack and just going back to running in ascending order everywhere, and going back to sweeping this problem under the rug -- it's just too damn hard, already cost me a week.

Reply:
moved to #350, dropped the commit here.